docs/ContentTypeParser.md

```diff
@@ -1,11 +1,11 @@
 <h1 align="center">Fastify</h1>
 
 ## Content Type Parser
```

Comment:
There should be some information in this document about how the parsed content is made available in the request handler. It is not obvious from the documentation that the handler should expect the parsed content to be `request.body`.